package.json

```diff
@@ -41,7 +41,7 @@
   "author": "Fullstack Academy of Code",
   "license": "MIT",
   "dependencies": {
-    "axios": "^0.19.0",
```

Comment:
Have you changed these dependencies for some reason?

Reply:
My changes can be deleted